parfive/config.py

```diff
@@ -35,7 +35,7 @@ def _default_aiohttp_session(config: "SessionConfig") -> aiohttp.ClientSession:
     `aiohttp.ClientSession` expects to be instantiated in a asyncio context
     where it can get a running loop.
     """
-    return aiohttp.ClientSession(headers=config.headers)
+    return aiohttp.ClientSession(headers=config.headers, requote_redirect_url=False)
```

Comment:
We should add a minimum version of aiohttp just to be safe (tbh we should already have one and I don't know if this is actually the thing that constrains us).

Reply:
Agreed, but you do not have any versioning of dependencies in this project, yet. So for this PR I'd suggest to continue without. And in parallel, what do you think about migrating this project to poetry? It's pretty straightforward, and I am happy to help with a PR.

Reply:
I would rather stick to setuptools for now, although I would be happy to move all the config into `pyproject.toml`.